test/functional/page_objects/dashboard_page_controls.ts

```diff
@@ -428,9 +428,20 @@ export class DashboardPageControls extends FtrService {
     await this.retry.try(async () => {
       expect(await this.optionsListPopoverGetAvailableOptions()).to.eql(expectation);
     });
+    if (await this.testSubjects.exists('optionsList__cardinalityBadge')) {
+      expect(await this.optionsListGetCardinalityBadgeValue()).to.be(
+        Object.keys(expectation.suggestions).length.toLocaleString()
+      );
+    }
```

Comment:
I figured it would make sense to test the badge as part of every `ensureAvailableOptionsEqual` rather than going into each search test to verify that the badge has the expected value :+1: